<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="groupview">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>데이터 로깅 시작</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>231</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QRadioButton" name="radioButton_14">
              <property name="text">
               <string>즉시 시작 버튼 사용</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_15">
              <property name="text">
               <string>특정 시간에:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit"/>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_16">
              <property name="text">
               <string>외부 트리거 시</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>50</y>
             <width>81</width>
             <height>111</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>DCV</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_4">
              <property name="text">
               <string>DCI</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Setting</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>96</width>
             <height>151</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Range DCV:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Range DCI:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>10</y>
             <width>96</width>
             <height>121</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>Select</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100mV</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1V</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10V</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100V</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1000V</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Auto</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>Select</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100mA</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1000A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Auto</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>샘플링 간격</string>
          </property>
          <widget class="QWidget" name="layoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>300</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QSpinBox" name="spinBox_6"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_7"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_8"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_9"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_10"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>301</width>
             <height>17</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>DD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>HH</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>MM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>SS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>MS</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>41</y>
             <width>117</width>
             <height>47</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>최소(100MS)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Custom</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>데이터 로깅 중지</string>
          </property>
          <widget class="QRadioButton" name="radioButton_7">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>211</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>지정한 수의 샘플 수집 후</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>111</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>경과 시간 후</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_8">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>151</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>즉시 중지버튼 사용</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>130</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>80</y>
             <width>195</width>
             <height>45</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>DD</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>HH</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>MM</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>SS</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="spinBox_18"/>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_16"/>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="spinBox_15"/>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="spinBox_17"/>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>150</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="mypyqtgraph" name="" native="true">
        <layout class="QHBoxLayout" name="graphview">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>재생</string>
    </property>
    <addaction name="actionStart"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>정지</string>
    </property>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>설정</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDCV">
   <property name="text">
    <string>DCV</string>
   </property>
  </action>
  <action name="actionDCV_Stop">
   <property name="text">
    <string>DCV_Stop</string>
   </property>
  </action>
  <action name="actionAll_Stop">
   <property name="text">
    <string>All_Stop</string>
   </property>
  </action>
  <action name="actionAll_Start">
   <property name="text">
    <string>All_Start</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mypyqtgraph</class>
   <extends>QWidget</extends>
   <header>mypyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
